lib/index.js

```diff
@@ -12,13 +12,15 @@ function AuthPassportLocal () {
   this.context = {};
   this.pipes = pipes;
 
-  this.init = function (config, context, isDummy) {
+  this.init = function (customConfig, context, isDummy) {
     var strategy;
+    const defaultConfig = {
+      'secret': 'changeme',
```

Comment:
Exactly my thought, as I've written before(https://github.com/kuzzleio/kuzzle-plugin-auth-passport-local/pull/28#discussion_r97745677)

As to who should generate the salt: my instinct tells me that it should be the 1st plugin instance to start, using a mutex to lock the salt resource until it is available to other plugin instances.